grazie/src/test/kotlin/org/rust/grazie/RsGrammarCheckingTest.kt

```diff
@@ -53,6 +53,18 @@ class RsGrammarCheckingTest : RsInspectionsTestBase(GrazieInspection::class) {
         }
     """, checkInComments = true)
 
+    // https://github.com/intellij-rust/intellij-rust/issues/7024
+    fun `test check single sentence in sequential comments`() = doTest("""
+        fn main() {
+            // Path to directory where someone <TYPO>write</TYPO>
+            // and from where someone reads
```

Comment:
Let's add a test with an empty line between comment. In this case, a `typo` should be emitted